Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,12 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.reduce((resultArr, item) => {
+    if (!resultArr.includes(item)) resultArr.push(item);
+    return resultArr;
+  }, []);
+  // Or making unique arrays using Set datatype
+  // return [...new Set(arr)];
```

Comment:
I would prefer using `Set` as it is much shorter and easier to understand 🙂

Reply:
I totally agree with you